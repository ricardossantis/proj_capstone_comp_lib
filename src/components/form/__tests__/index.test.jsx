import React from 'react'
import Form from '../index'
import renderer from 'react-test-renderer'

it('renders correctly', () => {
  const tree = renderer.create(<Form placeholder='digite aqui' />)

  expect(tree).toMatchSnapshot()
})
