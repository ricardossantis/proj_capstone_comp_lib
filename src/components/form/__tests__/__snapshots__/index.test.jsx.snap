// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<form>
  <input
    className="sc-bdnylx hkA-DPN"
    placeholder="digite aqui"
  />
</form>
`;
